internal/fnapi/userauth.go

```diff
@@ -29,15 +30,24 @@ type UserAuth struct {
 }
 
 func (user UserAuth) GenerateToken(ctx context.Context) (string, error) {
-	if user.Clerk != nil {
+	switch {
+	case os.Getenv("GITHUB_ACTIONS") == "true":
+		jwt, err := github.JWT(ctx, "")
```

Comment:
Let's add a parameter comment - or remove the parameter until we need it.

Reply:
Added a comment (I agree not nice to have just empty string there, but most likely we would need it, not sure tho)